apps/right-to-rent-check/translations/src/en/pages.json

```diff
@@ -1,5 +1,17 @@
 {
-  "name": {
-    "header": "Personal details"
+  "request-property": {
+    "header": "Does the person already live in the property?"
```

Comment:
Add this as the `legend` for the field, then remove the header. add legend: { className: 'visuallyhidden' } to the field config.